packages/visx-demo/src/sandboxes/visx-geo-albers-usa/Example.tsx

```diff
@@ -26,96 +26,125 @@ const { features: unitedStates } = topojson.feature(topology, topology.objects.s
   features: FeatureShape[];
 };
 
-export const colors: Array<string> = ['#744DCA', '#3D009C', '#9020FF', '#C630FD']
-
-const nudges = {
-  'FL': [11, 3],
-  'AK': [0, -4],
-  'CA': [-7, 0],
-  'NY': [5, 0],
-  'MI': [13, 20],
-  'LA': [-10, -3],
-  'HI': [-10, 10],
-  'ID': [0, 10],
-  'WV': [-2, 4],
-  'KY': [10, 0],
-  'TN': [0, 4]
-}
+export const colors: string[] = ['#744DCA', '#3D009C', '#9020FF', '#C630FD'];
+
+// X and Y adjustments to individual states
+const coordOffsets: Record<string, number[]> = {
+  FL: [11, 3],
+  AK: [0, -4],
+  CA: [-7, 0],
+  NY: [5, 0],
+  MI: [13, 20],
+  LA: [-10, -3],
+  HI: [-10, 10],
+  ID: [0, 10],
+  WV: [-2, 4],
+  KY: [10, 0],
+  TN: [0, 4],
+};
+
+/**
+ * These states are too small to have text labels 
+ * inside of them and are usually displayed with pointers.
+ * For simplicity they are omitted from this demo.
+ */
+const ignoredStates = ['VT', 'NH', 'MA', 'RI', 'CT', 'NJ', 'DE', 'MD'];
 
 const GeoAlbersUsa = ({ width, height, fullSize = true }: GeoAlbersUsaProps) => {
   const centerX = width / 2;
   const centerY = height / 2;
   const scale = (width + height) / 1.55;
 
-  const [ displayLabels, setDisplayLabels ] = useState(fullSize)
+  const [displayLabels, setDisplayLabels] = useState(fullSize);
 
   return width < 10 ? null : (
     <>
-      <svg width={width} height={height} style={{background, borderRadius: '14px'}}>
+      <svg width={width} height={height} style={{ background, borderRadius: '14px' }}>
         <AlbersUsa<FeatureShape>
           data={unitedStates}
           scale={scale}
           translate={[centerX, centerY - 25]}
         >
-          {({ features }) => features.map(({ feature, path, projection }, i) => {
-            let coords = projection(geoCentroid(feature))
-            let abbr:string = stateAbbrs[feature.id]
+          {({ features }) =>
+            features.map(({ feature, path, projection }, i) => {
+              const coords:[number,number] | null = projection(geoCentroid(feature));
```

Comment:
nit – might fail prettier/linting
```suggestion
              const coords:[number, number] | null = projection(geoCentroid(feature));
```